app/controllers/licences.controller.js

```diff
@@ -91,9 +92,16 @@ async function viewReturns (request, h) {
   })
 }
 
+async function submitSupplementaryFlag (request, h) {
+  await CheckSupplementaryBillingFlagService.go(request.payload)
+
+  return h.response().code(204)
+}
```

Comment:
It would be worth treating this as a background process.

By this, I mean the legacy service does not have to `await` the flagging to complete. For an example, check the controllers (and their respective backing services) in `/jobs`. Or ping me if you need further info.